electrical/calibration.MD

```diff
@@ -0,0 +1,183 @@
+		* As opposed to the drive motors, the corners use absolute encoders that are defined in one direction only, so you cannot switch the direction of the encoder
+
+Clipping would be where you go past the 360 deg mark on the encoder and it starts again at 0
+
+If you are getting encoder values of close to 2000 and the value stays above 2000 and does not reset to 0 while the shaft is spinning, it means that the voltage divider board is not working properly. Check to make sure the resistors on the voltage divider board are the correct values
+
+* Spin the encoder until it reads a value of roughly half its max value and reattach the gear, coupling it back to the rover.
+
+	(e) For each motor M1 and M2, we now need to obtain what is the new min and max encoder values. Using the PWM Settings tab, move the motors until they run into the physical hard stops. Record the min and max encoder values for each, but for the min values add at least 15 and the max subtract at least 155.
+	
+	(f) Navigate to the Position Settings tab.
+
+![](../images/calibration/is9.png)
+**Figure 9**
+
+	(g) Enter your Min and Max encoder values for each M1 and M2 motor
+
+We add a buffer of at least 5 encoder ticks to avoid driving the motors all the way into the hard stop; In other words, we’ll impose a software ’soft stop’
+ 
+![](../images/calibration/is10.png)  
+     **Figure 10**
+     
+	(h) Change the Autotune type to PD and then tune the M1 and M2 motors by clicking the ’Tune M1’ and ’Tune M2’ buttons. We’ve found that this autotuning process is more successful if you run it with the wheels down on a smooth surface, but still supporting the weight of the robot.
+
+![](../images/calibration/is11.png)
+**Figure 11**
+
+	(i) To verify that the corner steering motors have been calibrated, move the M1 and M2 sliders to their middle positions (halfway between min and max) and verify that the wheels all point exactly straight forwards. If they do not point straight forwards, verify that your min and max values are correct and ensure that no signal clipping is happening on the encoders.
+
+	(j) After the motors have both been tuned go to General settings and set the Motor current Max to be 3.0 Amps for each motor.
+	(k) Repeat Step 9 for the remaining corner motors
+
+## 2. Manual Calibration
+This section is to preform a manual tuning and calibration of the Corner motor system. This will be using the Ziegler-Nichols tuning method. For a more in depth explaination of how this process works you can visit:
+
+* [Tuning for PID Controllers](http://faculty.mercer.edu/jenkins_he/documents/TuningforPIDControllers.pdf)
+
+These steps assume you have already set the max/min values of the encoders in the position tab, as well as verified the encoder direction. If you have not done those follow the steps in the Corner Calibration
```

Comment:
i would bold this like in the original pdf

Reply:
???

Reply:
done